test/functional/feature_asmap.py

```diff
@@ -66,18 +67,27 @@ def test_asmap_with_relative_path(self):
     def test_default_asmap(self):
         shutil.copyfile(self.asmap_raw, self.default_asmap)
         for arg in ['-asmap', '-asmap=']:
-            self.log.info('Test bitcoind {} (using default map file)'.format(arg))
+            self.log.info('Test pivxd {} (using default map file)'.format(arg))
             self.stop_node(0)
             with self.node.assert_debug_log(expected_messages(self.default_asmap)):
                 self.start_node(0, [arg])
         os.remove(self.default_asmap)
 
     def test_default_asmap_with_missing_file(self):
-        self.log.info('Test bitcoind -asmap with missing default map file')
+        self.log.info('Test pivxd -asmap with missing default map file')
         self.stop_node(0)
         msg = "Error: Could not find asmap file '\"{}\"'".format(self.default_asmap)
         self.node.assert_start_raises_init_error(extra_args=['-asmap'], expected_msg=msg)
 
+    def test_empty_asmap(self):
+        self.log.info('Test pivxd -asmap with empty map file')
+        self.stop_node(0)
+        with open(self.default_asmap, "w", encoding="utf-8") as f:
+            f.write("")
+        msg = "Error: Could not parse asmap file \"{}\"".format(self.default_asmap)
+        self.node.assert_start_raises_init_error(extra_args=['-asmap'], expected_msg=msg)
```

Comment:
to fix this test case as well:
```suggestion
        self.assert_start_raises_init_error(0, extra_args=['-asmap'], expected_msg=msg)
```